resources/site-packages/xbmctorrent/scrapers/rutor.py

```diff
@@ -10,7 +10,7 @@
 from xbmctorrent.utils import ensure_fanart
 
 BASE_URL = "%s/" % plugin.get_setting("base_rutor")
-DOWNLOAD_URL = "http://d.rutor.org/"
+DOWNLOAD_URL = BASE_URL + "parse/d.rutor.org/"
```

Comment:
official mirrors don't use format like you mentioned `{mirror}/parse/d.rutor.org/{torrent Id}`, try as an example fast-bit dot org.